packages/shared/src/environment.ts

```diff
@@ -21,6 +21,9 @@ export const IS_APPLE: boolean =
 export const IS_FIREFOX: boolean =
   CAN_USE_DOM && /^(?!.*Seamonkey)(?=.*Firefox).*/i.test(navigator.userAgent);
 
+export const IS_ANDROID =
+  CAN_USE_DOM && /android/.test(navigator.userAgent.toLowerCase());
```

Comment:
Nit, you could've used case insensitive search just like FF above

```suggestion
  CAN_USE_DOM && /android/i.test(navigator.userAgent);
```